walk/walk_test.go

```diff
@@ -167,7 +167,7 @@ func TestCustomBuildName(t *testing.T) {
 		}
 	})
 	want := []string{
-		"sub/BUILD.test",
+		"sub/BUILD.bazel",
```

Comment:
Why would this happen? 🤔

Reply:
Oh, because `ValidBuildFileNames` can be changed with `# gazelle:build_file_name` in addition to via the `-build_file_name` cli arg 😢

Maybe that means this change is not possible?